format/Message.fbs

```diff
@@ -84,6 +84,10 @@ table Field {
   name: string;
   nullable: bool;
   type: Type;
+  // present only if the field is dictionary encoded
```

Comment:
I will add DictionaryBatch description. I vote for single dictionary per DictionaryBatch. This is simpler and the overhead of the DictionaryBatch header is minimal.